db/migration/1721137710001-RemoveHideLinesOutsideToleranceFromConfig.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class RemoveHideLinesOutsideToleranceFromConfig1721137710001
+    implements MigrationInterface
+{
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`-- sql
+            UPDATE charts
+            SET config = JSON_REMOVE(config, '$."hideLinesOutsideTolerance"')
+            WHERE
+                type = "ScatterPlot"
+            AND
+                slug IN (
+                  "stunting-vs-level-of-prosperity-over-time",
+                  "growth-of-income-and-trade"
+                )
+        `)
+    }
```

Comment:
You correctly bump the schema version in this PR (nice!) - but then to make it consistent we should also bump all existing schema versions in the charts table as part of this migration  (since the migration above makes sure that they actually adhere to the new schema)

Reply:
@danyx23 noted the positive feedback in the PR comment, I think this is a really good habit for devs in general, thanks :-)